cli/generic/forms/fields.py

```diff
@@ -186,3 +187,23 @@ def _deserialize(self, value, attr, data, **kwargs):
             raise ValidationError(f'The following swap identifier `{swap_identifier}` is invalid.')
 
         return swap_identifier
+
+
+class BlockIdentifierField(fields.Field):
```

Comment:
We have the same code in class `TransactionIdentifierField`, maybe we can just rename this class to `HeaderSignatureField` for example, and delete repeated code, what do you think, @dmytrostriletskyi?

Reply:
@Alladin9393, we will investigate the point of generic fields a bit later. For now, just leave the current approach.